lib/components/src/form/button.js

```diff
@@ -0,0 +1,12 @@
+import glamorous from 'glamorous';
+
+export default glamorous
+  .button({
+    border: 'solid 1px rgba(0, 0, 0, 0.2)',
+    background: 'rgba(255, 255, 255, 0.5)',
+    padding: '5px 10px',
```

Comment:
Let's add some more padding at the bottom, so that the text is vertically centered by lowercase letters. `4px 10px 7px` seems to be enough:
<img width="85" alt="screen shot 2017-11-11 at 03 59 28" src="https://user-images.githubusercontent.com/6651625/32684450-c7ac4f16-c694-11e7-8ed0-8028036c4300.png">